AlgorithmAnalyzer/Backend/tests/test_integration_endpoint_train.py

```
@@ -9,7 +9,14 @@
 from utils import SampleManager
 from main import app
 
-SAMPLE_UPLOAD_PATH = './data'
+try:
+    import config
+except ModuleNotFoundError:
```

Necessary?
Will silently fail and cause issues anyways later during the execution, just obscures where the code really fails